### PR TITLE
Add bBadger/USD, [bwBTC/ETH SLP]/USD, bDigg/USD and USD/bDigg as supported price identifiers

### PR DESCRIPTION
**Pull request by smb2796**

### CODE REVIEW

File: UMIPs/badger-price-ids.md [hunk 1]
```diff
@@ -0,0 +1,165 @@
+## HEADERS
+| UMIP-xyz     |                                                                                                                                  |
+|------------|------------------------------------------------------------------------------------------------------------------------------------------|
+| UMIP Title | Add bBadger/USD, [bwBTC/ETH SLP]/USD, bDigg/USD and USD/bDigg as supported price identifiers                                                                                                   |
```
Comment (pemulis) [inline UMIPs/badger-price-ids.md:4]: Are these price identifier strings going to cause havoc with the ExpressionPriceFeed? Would simplifying some of them to `bBadgerUSD`, `bDiggUSD`, and `USDbDigg` make them easier to work with? Not sure if the super-long identifier with the square brackets can be simplified without losing clarity. cc @mrice32

File: UMIPs/badger-price-ids.md [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+| Authors    | Sean Brown (smb2796) 
+| Status     | Approved                                                                                                                           |
```
Comment (pemulis) [inline UMIPs/badger-price-ids.md:6]: Isn't the status still `Draft`?

File: UMIPs/badger-price-ids.md [hunk 3]
```diff
@@ -0,0 +1,165 @@
...
+
+An additional use case that is being considered is creating call options on the value of Badger Vault tokens. For this reason, the USD denominated price identifiers for these vault tokens also need to be proposed. 
+
+Most methodologies required to calculate these price identifiers have already been defined in previous UMIPs. To avoid unnecessary redundancy, this UMIP will mostly reference that which has already been defined. 
+
+# MARKETS & DATA SOURCES
+
+- The markets and data sources for bBadger/USD and [bwBTC/ETH SLP]/USD remain the same as the ones listed in [UMIP-39](https://github.com/UMAprotocol/UMIPs/blob/master/UMIPs/umip-39.md) for USD/bBadger and USD-[bwBTC/ETH SLP] respectively.
+- To determine the underlying amount of DIGG for one bDIGG, the `pricePerFullShare` method needs to be queried on the bDIGG [contract](https://etherscan.io/address/0x7e7e112a68d8d2e221e11047a72ffc1065c38e1a). This information is on-chain and can be queried with any method that a voter wishes to use, since there should not be variation in the returned values.
```
Comment (pemulis) [inline UMIPs/badger-price-ids.md:31]: Any chance we have a subgraph for this?

File: UMIPs/badger-price-ids.md [hunk 4]
```diff
@@ -0,0 +1,165 @@
+## HEADERS
+| UMIP-xyz     |                                                                                                                                  |
+|------------|------------------------------------------------------------------------------------------------------------------------------------------|
+| UMIP Title | Add bBadger/USD, [bwBTC/ETH SLP]/USD, bDigg/USD and USD/bDigg as supported price identifiers                                                                                                   |
+| Authors    | Sean Brown (smb2796) 
+| Status     | Approved                                                                                                                           |
+| Created    | April 7th, 2021                                                                              
+
```
Comment (mhairimca) [inline UMIPs/badger-price-ids.md:8]: Discourse Link?

File: UMIPs/badger-price-ids.md (hunk 3 above)
Comment (smb2796) [reply UMIPs/badger-price-ids.md:31]: None that I know of, but will double check

File: UMIPs/badger-price-ids.md (hunk 2 above)
Comment (smb2796) [reply UMIPs/badger-price-ids.md:6]: Yep, this is a typo

File: UMIPs/badger-price-ids.md (hunk 1 above)
Comment (smb2796) [reply UMIPs/badger-price-ids.md]: Update the bDigg ones since they haven't been proposed before. Left the others for the sake of consistency.

**Review by mhairimca — APPROVED** [review]

**Pull request merged**